packages/next/src/server/image-optimizer.ts

```diff
@@ -98,8 +100,12 @@ export function getHash(items: (string | number | Buffer)[]) {
       hash.update(item)
     }
   }
-  // See https://en.wikipedia.org/wiki/Base64#Filenames
-  return hash.digest('base64').replace(/\//g, '-')
+  // See https://en.wikipedia.org/wiki/Base64#URL_applications
+  return hash.digest('base64url')
```

Comment:
Not necessary for the PR, but while the CACHE_VERSION is anyways changing, it doesn't hurt? 🤷

Reply:
I have considered this in the past but I don't know if its worth invalidating everyone's cache since there is no value being added here.

Reply:
Oh wait I just realized that you are adding upstreamEtag to the cache key 🤦‍♂️ 

So yes, now would be the time to change this. Looks like base64url was added in node@15.7.0 so we should be safe to use this now 👍